assets/xml/objects/object_drs.xml

```diff
@@ -1,20 +1,21 @@
-        <Texture Name="object_drs_Tex_004448" OutName="tex_004448" Format="i8" Width="32" Height="32" Offset="0x4448" />
-        <Texture Name="object_drs_Tex_004848" OutName="tex_004848" Format="rgba16" Width="32" Height="16" Offset="0x4848" />
-        <Texture Name="object_drs_Tex_004C48" OutName="tex_004C48" Format="rgba16" Width="16" Height="16" Offset="0x4C48" />
-        <Texture Name="object_drs_Tex_004E48" OutName="tex_004E48" Format="i8" Width="32" Height="64" Offset="0x4E48" />
-        <Texture Name="object_drs_Tex_005648" OutName="tex_005648" Format="i8" Width="32" Height="32" Offset="0x5648" />
-        <Limb Name="object_drs_Standardlimb_005A48" Type="Standard" Offset="0x5A48" />
-        <Limb Name="object_drs_Standardlimb_005A54" Type="Standard" Offset="0x5A54" />
-        <Limb Name="object_drs_Standardlimb_005A60" Type="Standard" Offset="0x5A60" />
-        <Skeleton Name="object_drs_Skel_005A78" Type="Flex" LimbType="Standard" Offset="0x5A78" />
+        <Animation Name="gWeddingDressMannequinIdleAnim" Offset="0x1C" />
+        <DList Name="gWeddingDressMannequinDL" Offset="0x3120" />
+        <DList Name="gWeddingDressMannequinPatternDL" Offset="0x3EA8" />
+        <DList Name="gWeddingDressMannequinMaskDL" Offset="0x3F80" />
+        <Texture Name="gWeddingDressMannequinWoodTex" OutName="wood" Format="rgba16" Width="16" Height="16" Offset="0x4008" />
+        <Texture Name="gWeddingDressMannequinWoodCornerTex" OutName="wood_corner" Format="rgba16" Width="16" Height="16" Offset="0x4208" />
+        <Texture Name="gWeddingDressMannequinShoulderGradientTex" OutName="drs_shoulder_gradient" Format="i8" Width="8" Height="8" Offset="0x4408" />
+        <Texture Name="gWeddingDressMannequinPatternTex" OutName="pattern" Format="i8" Width="32" Height="32" Offset="0x4448" />
+        <Texture Name="gWeddingDressMannequinVineTex" OutName="vine" Format="rgba16" Width="32" Height="16" Offset="0x4848" />
+        <Texture Name="gWeddingDressMannequinFlowerTex" OutName="flower" Format="rgba16" Width="16" Height="16" Offset="0x4C48" />
+        <Texture Name="gWeddingDressMannequinFabricHighlightsTex" OutName="fabric_highlight" Format="i8" Width="32" Height="64" Offset="0x4E48" />
+        <Texture Name="gWeddingDressMannequinFabricSheerTex" OutName="fabric_sheer" Format="i8" Width="32" Height="32" Offset="0x5648" />
+        <Limb Name="gWeddingDressMannequinLimb" Type="Standard" EnumName="WEDDING_DRESS_MANNEQUIN_LIMB" Offset="0x5A48" />
+        <Limb Name="gWeddingDressMannequinMaskLimb" Type="Standard" EnumName="WEDDING_DRESS_MANNEQUIN_MASK_LIMB" Offset="0x5A54" />
+        <Limb Name="gWeddingDressMannequinPatternLimb" Type="Standard" EnumName="WEDDING_DRESS_MANNEQUIN_PATTERN_LIMB" Offset="0x5A60" />
+        <Skeleton Name="gWeddingDressMannequinSkel" Type="Flex" LimbType="Standard" LimbNone="WEDDING_DRESS_MANNEQUIN_NONE_LIMB" LimbMax="WEDDING_DRESS_MANNEQUIN_MAX_LIMB" EnumName="WeddingDressMannequinLimbs" Offset="0x5A78" />
```

Comment:
The limb enum is of the form `{ACTOR_DESCRIPTION}_LIMB_{LIMB_NAME}` (i.e. `Limb` is only a suffix for the data in the asset file, not the enum.